content/pt/blog/2024/docs-localized.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Site do OpenTelemetry com suporte a múltiplas linguas!
+linkTitle: Site multilíngue
+date: 2024-08-10
+author: >-
+  [Severin Neumann](https://github.com/svrnm) (Cisco)
+issue: 4863
+sig: Comms
+default_lang_commit: 48eac183a4dd74946d5a45fa436cfc6052f30532
+---
+
+Estamos felizes em anunciar que o site do OpenTelemetry está disponível em
+múltiplas línguas! Os times de localização já começaram a traduzir páginas do
+site para [Chines](/zh), [Japonês](/ja), [Português](/pt), e [Espanhol](/es).
+
+O projeto OpenTelemetry cresceu para incluir contribuidores e usuários de todo o
+mundo. Criar um site disponível em múltiplas línguas é um passo importante para
+garantir que todos, independente da sua língua nativa, consigam contribuir para
+o projeto. Nos também estamos esperando que essa iniciativa conceda aos usuários
+finais acesso a documentações em diversas linguagens, facilitando o aprendizado
+e o entendimento do OpenTelemetry.
+
+Para acessar o site na sua língua preferida, utilize o seletor de linguagem no
+canto superior direito da página.
+
+Convidamos você para contribuir nesse trabalho. Você pode encontrar colegas que
+contribuem para localização no canal do Slack
+[#otel-docs-localization](https://cloud-native.slack.com/archives/C076RUAGP37).
+Se você é fluente em uma linguagem que já é suportada, você pode ajudar criando
+ou revisando traduções. Se sua língua ainda não é suportada e você gostaria de
+ajudar a traduzir,
+[crie uma issue](<https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Add+%3CYOUR%20LANGUAGE%3E+(%3CYOUR+CODE%3E)+version+of+website+pages&body=%3C!--+Provide+github+handles+of+at+least+2+people+that+will+work+on+this+translation+project%20--%3E>).
+
+Suas contribuições podem gerar um impacto significativo!
+
+Nos agradecemos a todos os contribuidores por fazer todo o possível, e nos
+estamos ansiosos para ver como essas novas opções de linguagens podem melhorar a
+experiência do usuário do OpenTelemetry.
```

Comment:
```suggestion
Nós agradecemos todas as pessoas que contribuíram para tornar isso uma realidade, e
estamos ansiosos para ver como essas novas opções de idioma podem melhorar a
experiência de usuário do OpenTelemetry.
```

"make this possible" é tipo "tornar realidade", não necessariamente fazer o possível.